app/graphql/mutations/organization_mutation.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Mutations
+  module OrganizationMutation
+    Update = GraphQL::Relay::Mutation.define do
+      name "updateOrganization"
+
+      input_field :name, types.String do
+        description 'Name of organization'
+      end
+
+      input_field :website, types.String do
+        description 'Website of organization'
+      end
+
+      return_type Types::OrganizationType
+
+      resolve ->(organization, args, _ctx) {
+        organization.update_attributes!(args.to_h)
+        organization
+      }
+    end
+  end
+end
```

Comment:
If you are thinking Rails admin views to manage the repo, we won't need this API as we will be creating max 20 orgs over the next few years using those views.  If you are thinking a JS admin page that uses this API, then nevermind :-)